lib/src/components/NxGlobalSidebar/__tests__/NxGlobalSidebarFooter.test.tsx

```diff
@@ -4,63 +4,131 @@
+      });
+
+      it('render default support text of "Help and Support" ' +
+      'if supportLink prop is provided but supportText is not', function() {
+        const view = quickRender({ supportLink: '#supporturl' }),
+            supportLink = view.getByRole('link');
+
+        expect(view.getByText('Help and Support')).toBeVisible();
+        expect(supportLink).toHaveTextContent(/^Help and Support$/);
+        expect(supportLink).toHaveAttribute('href', '#supporturl');
+      });
+    });
+
+    describe('releaseText', function() {
+      it('renders the Release div with a text and version number if releaseText is a string', function() {
+        const view = quickRender({ releaseText: 'React Shared Components: 3.1.4' });
+
+        expect(view.getByText('React Shared Components: 3.1.4')).toBeVisible();
+      });
+
+      it('renders the Release div with a text and version number if releaseText is jsx', function() {
+        const jsx =
+          <>
+            <span data-testid="spantext">TestSpan 3.1.4</span>
+          </>,
+            view = quickRender({ releaseText: jsx });
+
+        expect(view.getByTestId('spantext')).toBeInTheDocument();
+        expect(view.getByText('TestSpan 3.1.4')).toBeVisible();
+      });
+    });
+
+    describe('productTagLine', function() {
+      it('renders the Product name div with text if productTagLine is a string', function() {
+        const view = quickRender({ productTagLine: 'Powered by PLAID VILLAIN' });
+
+        expect(view.getByText('Powered by PLAID VILLAIN')).toBeVisible();
+      });
+
+      it('renders the Product name div with text if productTagLine is jsx', function() {
+        const jsx =
+          <>
+            <span data-testid="spantext">Powered by JSX</span>
+          </>,
+            view = quickRender({ productTagLine: jsx });
+
+        expect(view.getByTestId('spantext')).toBeInTheDocument();
+        expect(view.getByText('Powered by JSX')).toBeVisible();
+      });
+    });
+
+    it('renders all required texts when all additional supported props are provided', function() {
+      const view = quickRender({
+        supportText: 'Support for RSC',
+        supportLink: '#supporturl',
+        releaseText: 'React Shared Components: 3.1.4',
+        productTagLine: 'Powered by PLAID VILLAIN'
+      });
 
-  it('renders the Created By text by default and does not render when set to false', function() {
-    const createdByTrue = getShallowComponent().find('div.nx-global-sidebar__created-by'),
-        createdByFalse = getShallowComponent({ showCreatedBy: false }).find('div.nx-global-sidebar__created-by');
+      expect(view.getByText('Support for RSC')).toBeVisible();
+      expect(view.getByRole('link')).toHaveTextContent(/^Support for RSC$/);
+      expect(view.getByText('React Shared Components: 3.1.4')).toBeVisible();
+      expect(view.getByText('Powered by PLAID VILLAIN')).toBeVisible();
```

Comment:
Include the "Created by" line in this test as well

Reply:
> Include the "Created by" line in this test as well

Added here: 84425bca6c7f0554988dca551aed9fa25edeca2c